beacon-chain/rpc/beacon/attestations.go

```diff
@@ -249,6 +249,17 @@ func (bs *Server) StreamIndexedAttestations(
 					continue
 				}
 				bs.ReceivedAttestationsBuffer <- data.Attestation
+			} else if event.Type == operation.AggregatedAttReceived {
+				data, ok := event.Data.(*operation.AggregatedAttReceivedData)
+				if !ok {
+					// Got bad data over the stream.
```

Comment:
Can we log an error or warning? Seems like the stream could be easily broken and failing silently.

Reply:
sure. adding